src/shared/compatibility.js

```diff
@@ -1176,10 +1176,13 @@ PDFJS.compatibilityChecked = true;
   }
   WeakMap.prototype = {
     has(obj) {
+      if (typeof obj !== 'object' || obj === null) {
```

Comment:
Unless I'm misunderstanding the examples at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakMap#Examples, it appears that it should also work if `obj` is of type `function`.
So, do we need to add a `typeof obj !== 'function'` check as well here?